examples/data_preprocessing/mimic_dataset/prepare_mimic_cxr.ipynb

```diff
@@ -0,0 +1,1520 @@
+      "text/plain": [
+       "0          No Finding\n",
+       "1          No Finding\n",
+       "2          No Finding\n",
+       "3          No Finding\n",
+       "4       Consolidation\n",
+       "5          No Finding\n",
+       "6          No Finding\n",
+       "7    Pleural Effusion\n",
+       "Name: label, dtype: object"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "labels_chexpert['label'].head(8)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "100%|██████████| 227827/227827 [08:40<00:00, 437.50it/s]\n"
+     ]
+    }
+   ],
+   "source": [
+    "#change names to ids\n",
+    "labels = {'Atelectasis':0,\n",
+    "               'Cardiomegaly':1,\n",
+    "               'Consolidation':2,\n",
+    "               'Edema':3,\n",
+    "               'Enlarged Cardiomediastinum':4,\n",
+    "               'Fracture':5,\n",
+    "               'Lung Lesion':6,\n",
+    "               'Lung Opacity':7,\n",
+    "               'Pleural Effusion':8,\n",
+    "               'Pneumonia':9,\n",
+    "               'Pneumothorax':10,\n",
+    "               'Pleural Other':11,\n",
+    "               'Support Devices':12,\n",
+    "               'No Finding':13}        \n",
+    "        \n",
+    "for i in tqdm(range(len(labels_chexpert))):\n",
+    " labels_chexpert.iloc[i,16] = labels.get(labels_chexpert.iloc[i,16])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Merge labels and splits"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, the records, reports, labels and split are merged to one file. Note that sometimes one report refers to multiple X-rays (e.g. one frontal and one lateral X-ray) and since the labels are derived from the reports, one label might refer to multiple images."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#merge records, labels and split\n",
+    "record_report_split_list = pd.merge(record_report_list, split_list, how = 'left', on = ['dicom_id', 'study_id','subject_id'])\n",
+    "record_report_split_label_list = pd.merge(record_report_split_list, labels_chexpert.iloc[:,[0,1,16]], how = 'left', on = ['study_id','subject_id'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The train/validation/test split is taken from mimic-cxr-jpg, which is intended to provide a reference dataset split for studies using this data. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "train-val-test split proportions: split\n",
```

Comment:
statistics is good, but I think it also should be moved from there to mb the part where you download the reports